libs/ui/catalog/src/lib/organisation-preview/organisation-preview.component.html

```diff
@@ -4,7 +4,7 @@
     [title]="organisation.name"
   >
     <div
-      class="flex-shrink-0 bg-gray-100 rounded-lg overflow-hidden w-full border border-gray-300 h-36 group-hover:shadow-xl group-hover:border-0"
```

Comment:
Are we going to add some other animation here, besides the title color transition?

Reply:
I don't think that it's expected, it should be the same as result search items.